tailwind.config.js

```diff
@@ -8,6 +8,36 @@ module.exports = {
   theme: {
     extend: {
       colors: {
```

Comment:
**issue**: Looks good. Some time ago I introduced some more colors: `tertiary`, `success` & `warning`. Would be nice if you could also add those here and in the Customizing page.